charts/consul/values.yaml

```diff
@@ -32,6 +32,17 @@ global:
   adminPartitions:
     enabled: false
     name: "default"
+    # Partition service properties.
+    service:
+      # Annotations to apply to the partition service.
+      #
+      # ```yaml
+      # annotations: |
+      #   "annotation-key": "annotation-value"
+      # ```
+      #
+      # @type: string
```

Comment:
Is "string" correct as a type? wouldnt this be a map[string]string?

Reply:
We provide the input as a giant `string` as we use the `|` directive to begin. This is how annotations have been defined across the board in our values file TBH.